[DX-1058] gateway-release-5.2-amends (#4110)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md

#### Breaking Changes
This release has no breaking changes.

##### Early Access Features:

> Please note that the `Tyk OAS APIs` feature, currently marked as "Early Access", is subject to breaking changes in subsequent releases. Upgrading to a new version may introduce changes that are not backward-compatible. Downgrading to a previous version after upgrading may result in a broken installation. 

Reviewer: please add this link in all the prs https://tyk.io/docs/frequently-asked-questions/using-early-access-features/

> Please note that the `Tyk OAS APIs` feature, currently marked as "Early Access", is subject to breaking changes in subsequent releases. Upgrading to a new version may introduce changes that are not backward-compatible. Downgrading to a previous version after upgrading may result in a broken installation. 

Reviewer: ```suggestion
Please note that the `Tyk OAS APIs` feature, currently marked as "Early Access", is subject to breaking changes in subsequent releases. Upgrading to a new version may introduce changes that are not backwards-compatible. Downgrading or reverting an upgrade may not be possible result in a broken installation. 
```

> Users are strongly advised to follow the recommended upgrade instructions provided by Tyk before applying any updates.

Author: This is like a generic message, an eye opener, that Tyk OAS APIs is not production ready. The actual change wasn't introduced here

> Users are strongly advised to follow the recommended upgrade instructions provided by Tyk before applying any updates.

Reviewer 2: Ok, thanks @laurentiu, similar to a beta feature?

> Users are strongly advised to follow the recommended upgrade instructions provided by Tyk before applying any updates.

Author: yes

> Users are strongly advised to follow the recommended upgrade instructions provided by Tyk before applying any updates.

Reviewer: @lghiur @dcs3spp I think we should add "reverting" as Simon suggested. Just to be sure
